MiniBootcamp/MiniBootcamp/TweetModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  TweetModel.swift
+//  MiniBootcamp
+//
+//  Created by Chioma Amanda Mmegwa  on 29/05/2023.
+//
+
+import UIKit
+
+struct TweetCellViewModel {
+    let userName: String
+    let profileName: String
+    let profilePictureName: String
+    let content: String
+    
+    var profilePicture: UIImage? {
```

Comment:
It is always good to comment your code. I know it could be boring but that makes the difference when you work with legacy code. You could keep it simple like: /// The name of the user. I see you are using a good naming convention for your properties and that is great, sometimes writing this kind of in-line comment could seem a bit redundant. But it is a good practice that everyone needs to adopt.